keyboards/hidtech/bastyl/config.h

```diff
@@ -0,0 +1,41 @@
+/* 
+ * Copyright 2020 Quentin LEBASTARD <qlebastard@gmail.com>
+ * Copyright 2020 Anthony MARIN <anthony@hidtech.ca>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "config_common.h"
+#define VENDOR_ID 0x2F8A
+#define PRODUCT_ID 0x1827
+#define DEVICE_VER 0x0001
+#define MANUFACTURER HID Technologies
+#define PRODUCT Bastyl
+
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+#define DIODE_DIRECTION ROW2COL
+#define MATRIX_ROW_PINS { D7, B5, F7, F6, B6 }
+#define MATRIX_COL_PINS { B4, E6, C6, B1, B3, B2 }
+#define RGB_DI_PIN D2
+#define RGBLED_NUM 38
+#define RGBLIGHT_SPLIT
+#define RGBLIGHT_ANIMATIONS
+#define DEBOUNCE 5
+#define SOFT_SERIAL_PIN D0
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
+#define USE_SERIAL
+#define MASTER_RIGHT
```

Comment:
Don't need the serial define, actually.  And is there a reason for defining it as master right? 

(also, personally, I prefer "ee hands" since it allows both to work as master)

Reply:
Hi,

So the Serial is selected by default? Our keyboard uses Serial and not TRSS.
We defined it as Master Right by default a long time ago, partly because I had trouble getting ee_hands working correctly, and partly because we want consistency across our keyboards.